src/components/fetch-paginator.js

```diff
@@ -116,23 +143,18 @@ input:checked + poke-card {
         this.dispatchEvent(myEvent);
     }
 
-    async #handleClick(event) {
-        const target = event.target;
-        const { tagName, value } = target;
-        const IsButton = tagName === "BUTTON";
-        if (!IsButton) {
-            return;
+   #handleClick(value) {
+        return async () => {
+          const urlsToGoMap = new Map([
+              ["prev", this.#prevUrl],
+              ["home", this.#homeUrl],
+              ["next", this.#nextUrl]
+          ]);
+          const urlToGo = urlsToGoMap.get(value);
+          let checkboxs = this.shadowRoot.querySelectorAll('input[type=checkbox]');
+          Array.from(checkboxs).forEach(x => x.checked = false);
+          await this.#getDataOfTheRepository(urlToGo);
         }
-        const urlsToGoMap = new Map([
-            ["prev", this.#prevUrl],
-            ["home", this.#homeUrl],
-            ["next", this.#nextUrl]
-        ]);
-        const urlToGo = urlsToGoMap.get(value);
-        let checkboxs = this.shadowRoot.querySelectorAll('input[type=checkbox]');
-        Array.from(checkboxs).forEach(x => x.checked = false);
-        await this.#getDataOfTheRepository(urlToGo);
-        return;
```

Comment:
I need to change this because the `event.target` isn't working and the most easy way to fix it is that